File: src/main/java/io/github/hotlava03/baclava4j/commands/owner/OwnerCategory.java
package io.github.hotlava03.baclava4j.commands.owner;

import java.util.Arrays;
import java.util.List;

import discord4j.core.object.util.Permission;
import io.github.hotlava03.baclava4j.commands.Category;
import io.github.hotlava03.baclava4j.commands.CategoryType;
import io.github.hotlava03.baclava4j.commands.Command;

public class OwnerCategory implements Category {

    @Override
    public CategoryType getType() {
        return CategoryType.OWNER;
    }

    @Override
    public List<Command> getCommands() {
        return Arrays.asList(
            new Command(
                "jeval", // name
                new String[] {}, // aliases
                "Java quality eval only for lava", // description
                ">>jeval <code>", // usage
                new Permission[] {}, // permissions to use
                this, // Category
                new JEval() // the executor
        ));
    }

}

File: src/main/java/io/github/hotlava03/baclava4j/commands/owner/OwnerCategory.kt
package io.github.hotlava03.baclava4j.commands.owner

import io.github.hotlava03.baclava4j.commands.Category
import io.github.hotlava03.baclava4j.commands.CategoryType
import io.github.hotlava03.baclava4j.commands.Command

class OwnerCategory : Category {
    override val type: CategoryType
        get() = CategoryType.OWNER

    override val commands: List<Command>
        get() = listOf(
                Command(
                        "jeval", arrayOf(),  // aliases
                        "Java quality eval only for lava",  // description
                        ">>jeval <code>", arrayOf(),  // permissions to use
                        this,  // Category
                        JEval() // the executor
                ),
                Command(
                        "keval", arrayOf(),
                        "Kotlin quality eval only for lava",
                        ">>keval <code>", arrayOf(),
                        this,
                        KEval()
                )
        )
}